spec/requests/boxes_spec.rb

```diff
@@ -18,7 +18,49 @@
       box = JSON.parse(response.body)
       expect(response).to have_http_status(200)
       expect(box.length).to eq(1)
-      
+
+    end
+  end
+
+  describe "POST /create" do
+    it 'creates a new box' do
+
+      user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+      box_params = {
+        box: {
+            name: 'Dom Travis',
+            contents: 'Dumbells',
+            size: 'Medium',
+            user_id: user.id,
+        }
+      }
+      post '/boxes', params: box_params
+      expect(response).to have_http_status(200)
+
+      box = Box.first
+      expect(box.name).to eq('Dom Travis')
```

Comment:
Rspec has a cool matcher that lets you pass params. Check out `[have_attributes](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/have-attributes-matcher)`. You could do something like this: `expect(box).to have_attributes(box_params.box)